random_menu_generator_gold2.rb

```diff
@@ -0,0 +1,41 @@
+# outputs welcome
+puts "Welcome to Random Menu Generator!"
+print "How many meals would you like to see? "
+how_many = gets.chomp
+
+# outputs error when nil response or response does not include a number
+numbers = [ "0", "1", "2", "3", "4", "5", "6", "7", "8", "9" ]
+unless how_many.length != 0 && numbers.any? { |num| how_many.include?(num)}
```

Comment:
`numbers.any? { |num| how_many.include?(num)}` just checks that at least one character in `how_many` is a number, but that doesn't ensure they all are, so this would let something like `"seven8"` pass without the warning.  A similar approach might be `how_many.all? {|num| numbers.include?(num) }`.